tokens/color/color.brand.json

```
@@ -0,0 +1,37 @@
+{
+  "color": {
+    "brand": {
+      "$description": "Color used to express brand. Subject to theming. Use with caution!",
+      "main": {
+        "$description": "Loudest brand color. Currently reserved for header and tabs on public detail page.",
+        "value": "{color.palette.blue.base}",
+        "themeable":  true
+      },
+      "main-contrast": {
+        "$description": "Should pass AAA contrast check with the value of 'main'.",
+        "value": "{color.palette.white}",
+        "themeable":  true
+      },
+      "alt": {
+        "$description": "The second most prominent brand representation in the color scheme.",
+        "value": "{color.palette.teal.base}",
+        "themeable":  true
+      },
+      "alt-contrast": {
+        "$description": "Should pass AAA contrast check with the value of 'alt'.",
+        "value": "{color.palette.white}",
+        "themeable":  true
+      },
+      "highlight": {
+        "$description": "Brand accent color used for communicating active or focused state of interactive elements.",
```

This is simply not right anymore. 
```suggestion
        "$description": "Brand accent color to be used sparingly to raise awareness for important elements.",
```